config.rb

```diff
@@ -18,6 +24,37 @@
 tag = JSON.parse(response)['tag_name']
 set :latest_release_tag, tag
 
+# Populate docs pages
+docs_source_dir = 'kubeless-src'
+checkout_commit = '1c592cdc55bf009dc58773a042e42d70085e930b' # Use tag or a specific commit if necessary
+system "cd #{docs_source_dir} && git fetch origin && git checkout #{checkout_commit}"
+doc_files = Dir["#{docs_source_dir}/docs/*.md"]
+doc_files.each{ |f|
+  if !f.include? ".html.md" 
+    f_dir = File.dirname f
+    f_name = File.basename f, ".md"
+    if f_name != "README" # Avoid to parse the README file, it just contains a warning
```

Comment:
You can add this validation to the first guard clause. Check the previous comment.